metadata-ingestion/src/datahub/ingestion/source/hex/hex.py

```diff
@@ -93,9 +101,27 @@ class HexSourceConfig(
         default=True,
         description="Set ownership identity from owner/creator email",
     )
+    include_lineage: bool = Field(
+        default=True,
+        description="Include Hex lineage, being fetched from DataHub",
+    )
+    lineage_lookback_days: int = Field(
```

Comment:
we have some helpers for datetime and timedelta objects in configuration/datetimes.py - they might be useful here

Reply:
addressed in cf9c402000dead838c4fd55c0da3be1429a98e56

I considered on using BaseTimeWindowConfig but it sets a bucket field does not apply in the case of Hex as of today